src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -127,9 +127,12 @@ const TransferPanel = (): JSX.Element => {
               await latestToken.current.approve(selectedToken.address)
             }
           }
-          latestToken.current.deposit(selectedToken.address, amount)
+          const { decimals } = selectedToken as ERC20BridgeToken
```

Comment:
is this a safe cast?

Reply:
it is, but we can actually just type better to avoid having to., will do